recipe/meta.yaml

```
@@ -17,9 +17,9 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
-    - cython 0.29
-    - libxml2 2.10
-    - libxslt 1.1.37
+    - cython 3.0.10
+    - libxml2
```

```suggestion
    - libxml2 {{ libxml2 }} 
```
We want to align with other packages built against libxml2 and pin to the lowest version we plan to support.
libxml2 is set in the cbc file. Omitting the version would still use the one of the cbc, but this is more explicit.